### DISCUSSION TITLE
hdmi display like web browser

### DISCUSSION THREAD

**Discussion by surfninja2** [rern/rAudio#946] [General]
I have a monitor connected to the HDMI port.

Is it possible to display the songs, artist, ... like on a web browser on the HDMI output?

I currently get a login prompt.

RPI 1B, Wolfson DAC, rAudio 20220610.

**Comment by rern** [comment]
Settings > Features > Browser on RPi > Enable > Reboot

**Comment by surfninja2** [comment]
thanks!
here is my rpi2:
![image](https://user-images.githubusercontent.com/95239016/175052401-2d4ed59e-5037-42a3-b859-57647b1cc0d5.png)
system:
![image](https://user-images.githubusercontent.com/95239016/175053285-ee17f994-a083-4ad9-be1a-75c194991109.png)

here is my rpi 1B
![image](https://user-images.githubusercontent.com/95239016/175052727-7aa59ee5-bf47-4774-872f-22316bfb5e56.png)
system:
![image](https://user-images.githubusercontent.com/95239016/175053140-61649adb-4d18-4f0d-bf92-9fabc5e41bcd.png)


it might be hardware related (or the HDMI on the rpi 1B is not working correctly).

**Reply by surfninja2** [reply]
I looked at features.php and noticed it might be looking for /usr/bin/chromium.  this doesn't exist on my RP1
![image](https://user-images.githubusercontent.com/95239016/175085936-c3cf3b18-f8fe-4a1c-9d96-ccda311b950b.png)

i wonder if this is the problem.

i tried 
pacman -S chromium
error: target not found: chromium

chromium might not exist for my old RPi. oh well...

**Comment by rern** [comment]
Sorry, didn't notice it's RPi 1B. A browser is too much for `RPi 1` and `RPi Zero` to handle so it's not available.
